src/app-data/loaders/biologicLoader.ts

```diff
@@ -1,56 +1,72 @@
-import { v4 } from '@lukeed/uuid';
```

Comment:
Why remove the id generation?

Reply:
It is done by the template, as most loaders are returning the same data built from the `file`, I can remove it if it makes unclear.

Reply:
I see. The `templateFromFile` name confused me. Could you rename it to something more descriptive of what it does? Maybe `getMeasurementInfoFromFile()` ?

Reply:
Yes. Shall something this simple be tested separately? I thought it would not be necessary but I dont mind to add a `utility/__tests__`

Reply:
Yes. We should test as much as possible, especially things that seem the most simple, as they usually are simple to test too.